doc/conf.py.in

```diff
@@ -324,7 +324,19 @@ linkcheck_ignore = [
     # might not exist yet (we are generating it!)
     'https://docs.pgrouting.org/3.2/en/pgr_dijkstraNear.html',
     'https://docs.pgrouting.org/3.2/en/pgr_dijkstraNearCost.html',
+    'https://docs.pgrouting.org/3.2/en/pgr_randomSpanTree.html',
     'https://docs.pgrouting.org/lastes/en/pgr_makeConnected.html',
+    'https://docs.pgrouting.org/latest/en/coloring-family.html',
+    'https://docs.pgrouting.org/latest/en/pgr_bipartite.html',
+    'https://docs.pgrouting.org/latest/en/pgr_sequentialVertexColoring.html',
+    'https://docs.pgrouting.org/latest/en/pgr_dijkstraNear.html',
+    'https://docs.pgrouting.org/latest/en/pgr_dijkstraNearCost.html',
+    'https://docs.pgrouting.org/latest/en/pgr_lengauerTarjanDominatorTree.html',
+    'https://docs.pgrouting.org/latest/en/pgr_isPlanar.html',
+    'https://docs.pgrouting.org/latest/en/pgr_randomSpanTree.html',
+    'https://docs.pgrouting.org/latest/en/pgr_depthFirstSearch.html',
+    'https://docs.pgrouting.org/latest/en/traversal-family.html'
```

Comment:
You are missing a `,` at the end

Reply:
This missing comma is generating to many errors, please fix so I can continue with the review